### ISSUE TITLE
nans produced during SVD fitting over right skewed large rating values

### ISSUE THREAD

**Issue by nitayalon** [NicolasHug/Surprise#296]
#### Description
When fitting SVD over positive skewed rating data (Q1 = Q2 = Q3 = 1, 95 percentile = 214, max = 660) with default values (lr_all, reg_all, etc) after 2 epochs the b_i, b_u p_u and q_i are all nan. When setting the learning rate to 0.0005 **or** winsorizing the rating data at 95 percentile, the problem is solved. 

#### Steps/Code to Reproduce
svd_algo = SVD()
svd_algo.fit(train_set)

#### Expected Results
bi,bu, pu, qi all numeric
#### Actual Results
bi,bu, pu, qi all nan
#### Versions

Windows-10-10.0.17134-SP0
Python 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit (AMD64)]
surprise 1.0.6

**Comment by zsyf102900** [comment]
> (and obviously the scale of the ratings)

YES, WHEN I  tune  learing_rating  to 0.005 and speed n_epochs ,pu qi  filled with numeric values (previously nan)
is there something with scaling ？i saw  you set ouput within scale

**Comment by ananiask8** [comment]
I have entries in the matrix which form a left-skewed distribution, bounded between (0, 1). I am using NMF, and I have tried many settings but always get as pu and qi. Any tips?
